pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -204,7 +215,15 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 			"This will be a fatal error by default starting in K8s v1.6! " +
 			"In the meantime, you can opt-in to making this a fatal error by enabling --experimental-fail-swap-on.")
 	}
+	var capacity = v1.ResourceList{}
+	if info, err := cadvisorInterface.MachineInfo(); err == nil {
```

Comment:
is cadvisor actually guaranteed to be ready at this point with this information?  this is called prior to cadvisor.Start() since its an injected dependency...